packaging/ubuntu-16.04/snapd.install

```diff
@@ -8,6 +8,7 @@ usr/bin/snap-update-ns /usr/lib/snapd/
 usr/bin/snapd /usr/lib/snapd/
 usr/bin/snap-seccomp /usr/lib/snapd/
 usr/bin/snap-recovery /usr/lib/snapd/
+usr/bin/snap-preseed /usr/lib/snapd/
```

Comment:
Is this required to be present in the classic package?

How is this going to be used?

Reply:
It is going to be used on classic ubuntu systems only when preparing ubuntu cloud images (at least for the foreseeable future), it will be run against a mounted image. So it's a very specialized use case. Incorporating it into snap command is a possiblity, but for the time being the command is going to be usable for a very specific usecase only.